pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "answerking-python"
+version = "0.1.0"
+description = ""
+authors = ["michael-osullivan <michael.osullivan@answerdigital.com>"]
```

Comment:
Not to steal your thunder but might be worth removing your name from the files

Reply:
This is done